src/examples/subtotal/subtotal.ts

```diff
@@ -121,7 +124,8 @@ const vis: Subtotal = {
               pivotRow[pivots[i]] = pivotValues[i]
             }
             for (const measure of measures) {
-              const value = row[measure.name][flatKey].html || row[measure.name][flatKey].value
+              const obj = row[measure.name][flatKey]
+              const value = obj.html ? LookerCharts.Utils.htmlForCell(obj) : obj.value
```

Comment:
Seems like you could factor this ternary out to keep it a bit DRYer?

```
const htmlForCell = cell => obj.html ? LookerCharts.Utils.htmlForCell(obj) : obj.value
// Then call local htmlForCell all over
const cellValue = htmlForCell(cell)
```

Reply:
Also, `obj` and `value` are not very descriptive variable names. If `htmlForCell` accepts a cell as an argument, then I think that `obj` is probably a `cell` and `value` is probably `cellValue`?

Reply:
Makes sense. Whoever made this originally seems to really like `obj` because it's everywhere